### ISSUE TITLE
Is it possible to implement some kind of error handling? [feature-request]

### ISSUE THREAD

**Issue by getvince** [oyooyo/keyble-mqtt#2]
Hey oyooyo,
I was already using your previous implementation on a raspberry pi and now switched to the mqtt implementation.
Great job - really like it and it's one of the most useful things in my smarthome.

Unfortunately, sometimes the issued command gets send out to the broker but the lock doesn’t move. There seems to be no error handling feedback back to the broker or am I missing something? Would be quite useful so we could implement a retry mechanism.
Maybe even some kind of configuration within the keyble-mqtt client allowing to set the number of retries if the issued command times out or issues an error? Would be very handy!

Thanks a lot for restarting the development and implementing new features.

**Comment by oyooyo** [comment]
You're correct. Right now, there is pretty much no error handling at all, but of course there should be.

I have to admit I didn't even think about how to do error handling correctly so far, so if you have any suggestions how to do error handling in keyble-mqtt, let me know.
